src/fieri/app/controllers/fieri/jobs_controller.rb

```diff
@@ -20,5 +22,10 @@ def job_params
       end
       params
     end
+
+    def cookbook_version_response
+      uri = URI.parse("#{ENV['FIERI_SUPERMARKET_ENDPOINT']}/api/v1/cookbooks/#{params['cookbook_name']}/versions/#{params['cookbook_version']}")
+      Net::HTTP.get(uri)
+    end
```

Comment:
I thought through that...and I think we have more QM coming down the line which will need the cookbook version json.  I'd rather request it once here, then pass it to the worker.  In the future, I would like to refactor the metrics which operation on the cookbook json to request it in the controller and pass it to the worker.